goth_tests/domain/ya-provider/test_provider_multi_activity.py

```diff
@@ -23,14 +20,9 @@
 logger = logging.getLogger("goth.test.multi-activity")
 
 
-@pytest.mark.asyncio
-async def test_provider_multi_activity(
-    common_assets: Path,
-    config_overrides: List[Override],
-    log_dir: Path,
-):
-    """Test provider handling multiple activities in single Agreement."""
-
+def _create_runner(
+    common_assets: Path, config_overrides: List[Override], log_dir: Path
+) -> Tuple[Runner, Configuration]:
     nodes = [
         {"name": "requestor", "type": "Requestor"},
         {"name": "provider-1", "type": "VM-Wasm-Provider", "use-proxy": True},
```

Comment:
This override is no longer needed since you've created a custom config file (`goth_tests/domain/ya-provider/assets/goth-config.yml`). However, that file is not being used at the moment!

in the current implementation (line 32 in this revision, I cannot reach it with a PR comment)  `common_assets` is used as the base path for the config file. This `common_assets` is provided by a fixture and always resolves to `goth_tests/assets` (relative to the project root).
Instead, to use the "local" config file this should look something like this:
```
    goth_config = load_yaml(Path(__file__).parent / "assets" / "goth-config.yml", config_overrides)
```